ipykernel/debugger.py

```diff
@@ -270,13 +270,14 @@ class Debugger:
         'richInspectVariables', 'modules'
     ]
 
-    def __init__(self, log, debugpy_stream, event_callback, shell_socket, session):
```

Comment:
Can we make this an optional arg so it doesn't change the API?

Reply:
@blink1073 Thx for the review. I have just pushed a change for that.